charts/ratify/templates/secret-role-binding.yaml

```diff
@@ -0,0 +1,35 @@
+{{- if .Values.rbac.create }}
```

Comment:
Can we add a comment to the templates so we can distinguish this role for secret reading from the cluster role for secret reading needed for k8s auth provider in `roles.yaml`?

Reply:
that's good point, actually only cert-controller needs extra permission for listing/updating. I'll put it behind the cert-rotator featureFlag as well.